lib/ansible/module_utils/network/iosxr/providers/cli/config/base.py

```diff
@@ -0,0 +1,77 @@
+#
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+from ansible.module_utils.six import iteritems
+from ansible.module_utils.network.common.utils import to_list
+from ansible.module_utils.network.common.config import NetworkConfig
+
+
+class ConfigBase(object):
```

Comment:
What function does this class perform? It does not seem to be used. This class is also included in your other BGP series modules (ios _bgp, eos _bgp, iosxr _bgp, frr _bgp). I looked at the development history of the module ios _bgp and it looks like it was used previously, but not now. And in the history of this module there are no signs of its use at all.

Reply:
@hexdump0x0200 You're correct. Thanks for pointing this out. This unwillingly made its way into the PRs.